paper/paper.md

```diff
@@ -31,17 +31,40 @@ bibliography: paper.bib
+Analyzing and identifying genomic variants is particularly helpful when linking the _disease onset_ to the genomic predictions. Genomic variants play a key role in the identification and 
+characterization of the _disease onset_ and also allow for interlinking genomic variability and hypothesis testing. eVai, a variant annotation platform, provides annotation of the genomic variants 
+using several backend open source databases and tools. eVaiutilities proposed in this paper, is a ~~post-eVAI~~ analyzer for population scale analysis for eVAI variant annotation. It allows the 
+_analysis of the genomic variants further such as analyzing the multiple genomic annotated variants_, reference and alternate allele, enabling coordinate search, coordinate search with specified 
+variants and annotation search across _a large number of population_. The availability of the command line parameters allows for a large scale analysis across the several population files. _eVAI 
+has been used by many people and has been very well cited_, however, it does not allow for any high-throughput analyses. Thus, we developed eVaiutilities, command-line tool that enables many 
+analyses at a time, and has more functionality than eVai.
 
 # Statement of need
 
-Human genomics has always been an intriguing and gleaming topic of interest. Many efforts have been laid for understanding and identifying variants. Apart from the open source genomics efforts, proprietary efforts have been developed for the identification and annotation of the variants. The identified variants have been classified and linked to genetic and metabolic disorders, with severity ranging from pathogenic to highly pathogenic. One such effort is the eVai https://www.engenome.com/product/, which is a high throughput variant annotator, and provides the hierarchical classification of the genomic variants. However, the limitation of the eVai output files is the information spanning across the multiple files and the missing of the genome information and sub-sequence analysis on the annotated variants. eVaiutilities allows, you to use the output from the eVai and performs genome scale analysis of the eVai annotated variants. It allows you to search across populations, variants and spanning information across multiple samples. It is encoded in RUST, which is a memory safe language and provides scalability and uses RUST version: rustc 1.89.0.
+Human genomics has always been an intriguing and gleaming topic of interest. Many efforts have been laid for understanding and identifying variants. Apart from the _open source genomics efforts_,
+proprietary efforts have been developed for the identification and annotation of the variants. _The identified variants have been classified and linked to genetic and metabolic disorders_, with 
```

Comment:
Again - too general statement, please add examples, references

Reply:
Please, rewrite the sentence then, if you cannot give a reference to a general paper.